### ISSUE TITLE
Cannot install master version of cookiecutter

### ISSUE THREAD

**Issue by mathisc** [cookiecutter/cookiecutter#1614]
* Python version: Python 3.8.10
* Operating System: Linux pop-os 5.13.0-7614-generic

### Description:
Running `pip install git+https://github.com/cookiecutter/cookiecutter.git` leads to the following error : 
```bash
Collecting git+https://github.com/cookiecutter/cookiecutter.git
  Cloning https://github.com/cookiecutter/cookiecutter.git to /tmp/pip-req-build-75pdjo4d
  Running command git clone --filter=blob:none -q https://github.com/cookiecutter/cookiecutter.git /tmp/pip-req-build-75pdjo4d
  Resolved https://github.com/cookiecutter/cookiecutter.git to commit d6037b7dee5756e35a6ecd5b522899a9061c2c79
  Running command git submodule update --init --recursive -q
  Installing build dependencies ... done
  Getting requirements to build wheel ... done
  Preparing metadata (pyproject.toml) ... error
  ERROR: Command errored out with exit status 1:
   command: /home/claire/.virtualenvs/base/bin/python /home/claire/.virtualenvs/base/lib/python3.8/site-packages/pip/_vendor/pep517/in_process/_in_process.py prepare_metadata_for_build_wheel /tmp/tmpdy_3ooya
       cwd: /tmp/pip-req-build-75pdjo4d
  Complete output (37 lines):
  running dist_info
  creating /tmp/pip-modern-metadata-fkrvafv1/cookiecutter.egg-info
  writing /tmp/pip-modern-metadata-fkrvafv1/cookiecutter.egg-info/PKG-INFO
  Traceback (most recent call last):
    File "/home/claire/.virtualenvs/base/lib/python3.8/site-packages/pip/_vendor/pep517/in_process/_in_process.py", line 363, in <module>
      main()
    File "/home/claire/.virtualenvs/base/lib/python3.8/site-packages/pip/_vendor/pep517/in_process/_in_process.py", line 345, in main
      json_out['return_val'] = hook(**hook_input['kwargs'])
    File "/home/claire/.virtualenvs/base/lib/python3.8/site-packages/pip/_vendor/pep517/in_process/_in_process.py", line 164, in prepare_metadata_for_build_wheel
      return hook(metadata_directory, config_settings)
    File "/tmp/pip-build-env-6rsf0ae5/overlay/lib/python3.8/site-packages/setuptools/build_meta.py", line 174, in prepare_metadata_for_build_wheel
      self.run_setup()
    File "/tmp/pip-build-env-6rsf0ae5/overlay/lib/python3.8/site-packages/setuptools/build_meta.py", line 158, in run_setup
      exec(compile(code, __file__, 'exec'), locals())
    File "setup.py", line 10, in <module>
      setuptools.setup(
    File "/tmp/pip-build-env-6rsf0ae5/overlay/lib/python3.8/site-packages/setuptools/__init__.py", line 153, in setup
      return distutils.core.setup(**attrs)
    File "/usr/lib/python3.8/distutils/core.py", line 148, in setup
      dist.run_commands()
    File "/usr/lib/python3.8/distutils/dist.py", line 966, in run_commands
      self.run_command(cmd)
    File "/usr/lib/python3.8/distutils/dist.py", line 985, in run_command
      cmd_obj.run()
    File "/tmp/pip-build-env-6rsf0ae5/overlay/lib/python3.8/site-packages/setuptools/command/dist_info.py", line 31, in run
      egg_info.run()
    File "/tmp/pip-build-env-6rsf0ae5/overlay/lib/python3.8/site-packages/setuptools/command/egg_info.py", line 292, in run
      writer(self, ep.name, os.path.join(self.egg_info, ep.name))
    File "/tmp/pip-build-env-6rsf0ae5/overlay/lib/python3.8/site-packages/setuptools/command/egg_info.py", line 656, in write_pkg_info
      metadata.write_pkg_info(cmd.egg_info)
    File "/usr/lib/python3.8/distutils/dist.py", line 1117, in write_pkg_info
      self.write_pkg_file(pkg_info)
    File "/tmp/pip-build-env-6rsf0ae5/overlay/lib/python3.8/site-packages/setuptools/dist.py", line 167, in write_pkg_file
      write_field('Summary', single_line(self.get_description()))
    File "/tmp/pip-build-env-6rsf0ae5/overlay/lib/python3.8/site-packages/setuptools/dist.py", line 151, in single_line
      raise ValueError('Newlines are not allowed')
  ValueError: Newlines are not allowed
  ----------------------------------------
```

It looks like the commit that creates the issue on master is  `f00bed7b93aff1a7684bfb4dbba287999920b76f ` (installing a version below that commit works fine for me).

Thanks a lot,

**Comment by Tonow** [comment]
I try this https://github.com/cookiecutter/cookiecutter/pull/1615

**Comment by jurgenwigg** [comment]
Hi! Tried this on `Linux Jurgen 5.15.0-27-generic #28-Ubuntu SMP Thu Apr 14 04:55:28 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux` with `Python 3.8.13`. Scenario:
1. Freshly created venv
2. Activated created venv
3. Executed provided command
Everything executed smoothly without any issues. Maybe for this moment this is not an issue anymore?

**Comment by Tonow** [comment]
@jurgenwigg :+1:  
I think it's cause the https://github.com/cookiecutter/cookiecutter/blob/master/setup.cfg is now fixed ( I propose that [#1615](https://github.com/cookiecutter/cookiecutter/pull/1615) ) but thanks to  @ozer550  for one bigger clean :broom: 
https://github.com/cookiecutter/cookiecutter/commit/db5fce2ee25b5ece417c12dddfc712beaace61fe#diff-fa602a8a75dc9dcc92261bac5f533c2a85e34fcceaff63b3a3a81d9acde2fc52

I will close my PR @mathisc  if it's ok for you may be you can close this issue

**Comment by jensens** [comment]
It seems this one was fixed. Please reopen if I am wrong.
